commit 7e826f473978cdabcf75b57df77ad78f4e0fc815
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-27T14:51:05-07:00 (3 months ago)
    
    TRITON-1779 sapi mode.test.js creates cpu_cap-less VMs

diff --git a/test/common.js b/test/common.js
index a6cfd0d..3bd64d4 100644
--- a/test/common.js
+++ b/test/common.js
@@ -53,6 +53,7 @@ function createService(app_uuid, uuid, cb) {
     var opts = {};
     opts.params = {};
     opts.params.ram = 256;
+    opts.params.cpu_cap = 100;
     opts.params.networks = [ 'admin' ];
     assert.string(process.env.SAPI_TEST_IMAGE_UUID,
         'process.env.SAPI_TEST_IMAGE_UUID');
